lib/app.coffee

```diff
@@ -14,18 +14,20 @@ See the License for the specific language governing permissions and
 limitations under the License.
 ###
 
-Raven = require('raven')
-Raven.disableConsoleAlerts()
-Raven.config require('./config').sentryDsn,
+Analytics = require('analytics.node').core
+Raven = require('analytics.node').sentryIntegration
+dsn = require('./config').sentryDsn
+release = require('../package.json').version
+ravenOptions =
+	config: dsn
+	release: release
 	captureUnhandledRejections: true
-	release: require('../package.json').version
-.install (logged, error) ->
-	console.error(error)
-	process.exit(1)
-Raven.setContext
-	extra:
-		args: process.argv
-		node_version: process.version
+	disableConsoleAlerts: true
+Analytics.addIntegration(Raven)
+Analytics.initialize 'Sentry': ravenOptions
+Analytics.setContext Analytics.anonymize(extra:
+	args: process.argv
+	node_version: process.version)
```

Comment:
I think our general style guide for coffeescript is that you use brackets for one-line function calls, and no brackets for multiple line calls, so this shouldn't have brackets, and the line before should.

Reply:
Personally, I'd also drop the extra down a line, like:

```coffeescript
Analytics.setContext Analytics.anonymize
  extra:
    args: process.argv
    node_versionn: process.version
```

Reply:
Yup: https://github.com/resin-io/process/blob/master/process/coffeescript-coding-standards.md